packages/rest/src/validation/request-body.validator.ts

```diff
@@ -34,7 +34,6 @@ export function validateRequestBody(
   globalSchemas?: SchemasObject,
 ) {
   if (!requestBodySpec) return;
-
```

Comment:
Please preserve these empty lines, they serve as visual delimiter between different sections of this long(ish) method.